docs/content/docs/custom-resource/autoscaler.md

```diff
@@ -66,9 +66,9 @@ The defaults might work reasonably well for many applications, but some tuning m
 ### Job and per operator max parallelism
 
 When computing the scaled parallelism, the autoscaler always considers the max parallelism settings for each job vertex to ensure that it doesn't introduce unnecessary data skew.
-The computed parallelism will always be a divisor of the max_parallelism number.
+The computed parallelism will always be a divisor of the max parallelism number.
```

Comment:
Just to be clear: Max parallelism refers to either the default `pipeline.max-parallelism` setting or the max parallelism defined directly at the operator.